db/schema.rb

```diff
@@ -612,9 +612,10 @@
     t.string   "control_device_number", limit: 255
     t.datetime "created_at",                        null: false
     t.datetime "updated_at",                        null: false
+    t.string   "description",           limit: 255
   end
 
-  add_index "secure_rooms_card_readers", ["product_id"], name: "index_secure_rooms_card_readers_on_product_id", using: :btree
+  add_index "secure_rooms_card_readers", ["card_reader_number", "control_device_number"], name: "index_identifying_numbers", unique: true, using: :btree
```

Comment:
It's probably fine, but this index now seems overly general. What about something like `idx_secure_room_reader`?